modules/suggestions/reactions.ts

```diff
@@ -36,11 +37,56 @@ export async function addToDatabase(thread: AnyThreadChannel): Promise<void> {
 	const defaultEmoji = config.channels.suggestions?.defaultReactionEmoji;
 	const message = await thread.fetchStarterMessage().catch(() => void 0);
 	const count = (defaultEmoji?.id && message?.reactions.resolve(defaultEmoji.id)?.count) || 0;
+	const suggestionData = getSuggestionData(thread);
+	suggestionsDatabase.data = [...suggestionsDatabase.data, { ...suggestionData, count }];
+	const data = suggestionsDatabase.data as {
+		answer: typeof suggestionAnswers[number];
```

Comment:
This should work without an `as`

Reply:
ts throws a ` is 'readonly' and cannot be assigned to the mutable type `